infra/core/managed-cluster/aks-cluster.parameters.json

```diff
@@ -53,12 +54,19 @@
         "resourceGroup": "#{{ dnsResourceGroup }}"
       }
     },
-    "containerRegistry": {
-      "value": {
-        "name": "#{{ infraResourceNamePrefix }}#{{ nc_resource_containerregistry }}#{{ nc_instance_regionid }}01",
-        "resourceGroup": "#{{ servicesResourceGroup }}",
-        "subscriptionId": "#{{ subscriptionId }}"
-      }
+    "containerRegistries": {
+      "value": [
+        {
+          "name": "#{{ ssvSharedAcrName }}",
+          "resourceGroup": "#{{ ssvSharedResourceGroup }}",
+          "subscriptionId": "#{{ ssv3subscriptionId }}"
```

Comment:
Yes, it will be ssv5 subscription in defra environment. Its curretnly defind in the common varaibles files. Not sure how we going to switch to SSV5. We might conditioanly set this variable, Need to discuss on this.
@DaRoszko What do you think?

Reply:
I think we should move the variable to the environment file.

Reply:
Yeah I think it makes sense to move it to environment file or create a variable 'ssvSubscriptionId' which uses the value of either ssv3subscriptionId or ssv5subscriptionId from common depending on if it's a lower or higher environment.

Reply:
Added a new varaible in the enviroments files.